build/template-restore-build-libsandsamples.yaml

```diff
@@ -21,10 +21,9 @@ steps:
   inputs:
     solution: LibsAndSamples.sln
     vsVersion: '15.0'
-    msbuildArgs: '/p:RunCodeAnalysis=false /p:ReferencedBinsPathRoot=$(Build.BinariesDirectory)\FxCopRefAssemblies /p:MsalClientSemVer=${{ parameters.MsalClientSemVer }} /p:SourceLinkCreate=true'
```

Comment:
@bgavrilMS : don't we need the FxCop assemblies any longer?

Reply:
No, we use Roslyn analyzers. FxCop is dead.